django/db/backends/utils.py

```diff
@@ -194,20 +194,30 @@ def rev_typecast_decimal(d):
     return str(d)
 
 
-def truncate_name(name, length=None, hash_len=4):
+def split_identifier(identifier):
```

Comment:
I'm not sure about the whole _identifier_ naming here. I felt like we needed to distinguish between potentially namespaced names and naive ones. I'd be happy to use a more appropriate name given one is proposed or simply rename the function to `split_name` if deemed more appropriate.

Reply:
It could be useful to give an example in the docstring of what "identifier" might be.